bench/tx-generator/src/Cardano/Benchmarking/Script/Core.hs

```diff
@@ -306,12 +306,11 @@ localSubmitTx tx = do
   submitTracer <- btTxSubmit_ <$> get BenchTracers
   submit <- getLocalSubmitTx
   ret <- liftIO $ submit tx
-  let
-    msg = case ret of
-      SubmitSuccess -> "local submit success."
-      SubmitFail e -> mconcat
-        [ "local submit failed: " , show e , " (" , show tx , ")"]
-  liftIO $ traceWith submitTracer $ TraceBenchTxSubDebug msg
+  case ret of
+    SubmitSuccess -> return ()
+    SubmitFail e -> do
+      let msg = mconcat [ "local submit failed: " , show e , " (" , show tx , ")"]
+      liftIO $ traceWith submitTracer $ TraceBenchTxSubDebug msg
```

Comment:
Our typically accepted style here would be to avoid the `let`:
```
liftIO $ traceWith submitTracer $
  TraceBenchTxSubDebug $ mconcat
    [ "local submit failed: " , show e , " (" , show tx , ")"]
```